components/steps/icons.tsx

```diff
@@ -0,0 +1,150 @@
+    <svg viewBox='0 0 48 48' fill='none' xmlns='http://www.w3.org/2000/svg' {...props}>
+      <path
+        fillRule='evenodd'
+        clipRule='evenodd'
+        d='M8 11.185L20.7846 3.80385C22.641 2.73205 24.9282 2.73205 26.7846 3.80385L39.5692 11.185C41.4256 12.2568 42.5692 14.2376 42.5692 16.3812V31.1436C42.5692 33.2872 41.4256 35.2679 39.5692 36.3397L26.7846 43.7209C24.9282 44.7927 22.641 44.7927 20.7846 43.7209L8 36.3397C6.14359 35.2679 5 33.2872 5 31.1436V16.3812C5 14.2376 6.14359 12.2568 8 11.185ZM19.7846 2.0718C22.2598 0.642734 25.3094 0.642734 27.7846 2.0718L40.5692 9.45299C43.0444 10.8821 44.5692 13.5231 44.5692 16.3812V31.1436C44.5692 34.0017 43.0444 36.6427 40.5692 38.0718L27.7846 45.453C25.3094 46.8821 22.2598 46.8821 19.7846 45.453L7 38.0718C4.52479 36.6427 3 34.0017 3 31.1436V16.3812C3 13.5231 4.52479 10.8821 7 9.45299L19.7846 2.0718Z'
+        fill='#6075AA'
+      />
+      <g clipPath='url(#clip0_63_483)'>
+        <path
+          d='M14.7847 17.2499C16.0273 17.2499 17.0347 16.2426 17.0347 14.9999C17.0347 13.7573 16.0273 12.7499 14.7847 12.7499C13.542 12.7499 12.5347 13.7573 12.5347 14.9999C12.5347 16.2426 13.542 17.2499 14.7847 17.2499Z'
+          stroke='#6075AA'
+          strokeWidth='2'
+          strokeLinecap='round'
+          strokeLinejoin='round'
+        />
+        <path
+          d='M14.7847 26.2499C16.0273 26.2499 17.0347 25.2426 17.0347 23.9999C17.0347 22.7573 16.0273 21.7499 14.7847 21.7499C13.542 21.7499 12.5347 22.7573 12.5347 23.9999C12.5347 25.2426 13.542 26.2499 14.7847 26.2499Z'
+          stroke='#6075AA'
+          strokeWidth='2'
+          strokeLinecap='round'
+          strokeLinejoin='round'
+        />
+        <path
+          d='M14.7847 35.2499C16.0273 35.2499 17.0347 34.2426 17.0347 32.9999C17.0347 31.7573 16.0273 30.7499 14.7847 30.7499C13.542 30.7499 12.5347 31.7573 12.5347 32.9999C12.5347 34.2426 13.542 35.2499 14.7847 35.2499Z'
+          stroke='#6075AA'
+          strokeWidth='2'
+          strokeLinecap='round'
+          strokeLinejoin='round'
+        />
+        <path
+          d='M20.0347 14.9999H35.0347'
+          stroke='#6075AA'
+          strokeWidth='2'
+          strokeLinecap='round'
+          strokeLinejoin='round'
+        />
+        <path
+          d='M20.0347 23.9999H35.0347'
+          stroke='#6075AA'
+          strokeWidth='2'
+          strokeLinecap='round'
+          strokeLinejoin='round'
+        />
+        <path
+          d='M20.0347 32.9999H35.0347'
+          stroke='#6075AA'
+          strokeWidth='2'
+          strokeLinecap='round'
+          strokeLinejoin='round'
+        />
+      </g>
+      <defs>
+        <clipPath id='clip0_63_483'>
+          <rect width='24' height='24' fill='white' transform='translate(11.7847 11.9999)' />
+        </clipPath>
+      </defs>
+    </svg>
+  )
+}
+
+type UseStepIconResult = { [k: string]: string }
+
+export function useStepIcon(currentStepIdx: number, activeStepIdx: number): UseStepIconResult {
```

Comment:
This hook function is used in the Steps function while iterating over step icons.